Otec/Constants.swift

```diff
@@ -10,4 +10,6 @@ import Foundation
 //Application.settings.apiURL
 struct Constants {
     static let newspaperUserDefaultsKey = "userNewspaperID"
```

Comment:
Remember, we should use pure `CamelCase` (starting with uppercase) for `static` constants 😉